.github/workflows/test_suite.yaml

```diff
@@ -20,86 +20,85 @@ jobs:
       ENV_NAME: tests
       PYTHON: ${{ matrix.python-version }}
       OS: ${{ matrix.os }}
+      CC: gcc
```

Comment:
so for me, this wouldn't have worked... i couldn't get the OS to allow me to alias gcc to the non-clang version.  It is seems hardcoded to point to /usr/bin/gcc.  So I had to install by pre-pending the homebrew compiler version, i.e. gcc-11 that is in /usr/local/bin:

$ which gcc
/usr/bin/gcc
$ which gcc-11
/usr/local/bin/gcc-11

$ gcc --version
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/4.2.1
Apple clang version 13.0.0 (clang-1300.0.29.3)
Target: x86_64-apple-darwin20.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin

$ gcc-11 --version
gcc-11 (Homebrew GCC 11.2.0) 11.2.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.